builtin/common/serialize.lua

```diff
@@ -209,13 +218,15 @@ end
 -- Unit tests
 local test_in = {cat={sound="nyan", speed=400}, dog={sound="woof"}}
 local test_out = core.deserialize(core.serialize(test_in))
+local test_out = core.deserialize(core.serialize(test_in, true))
```

Comment:
you are just overwriting the old value, if you want to test these add a new unittest